tests/unit_tests/test_game_stats_service.py

```diff
@@ -114,9 +114,7 @@ def unit_stats():
     }
 
 
-async def test_process_game_stats(
-    game_stats_service, event_service, achievement_service, player, game
-):
+async def test_process_game_stats(game_stats_service, event_service, achievement_service, player, game):
```

Comment:
Keep lines shorter than 80 chars unless there is a good formatting reason.

Reply:
Wrapped the lines!

I will defend using an extra 4 spaces for the function arguments though, as it places them on a different indentation than the function's body.

Reply:
The 8 space indentation has come up before and we decided against it. I recently added a [CONTRIBUTING.md](https://github.com/FAForever/server/blob/develop/CONTRIBUTING.md) file with some style examples.

You don't necessarily need to put the arguments all on their own line for functions which will rarely change like these. It does make merging a lot easier when there are conflicts and it gives more space for typing.

Reply:
I've reverted the 8 space tabs, and cleaned up a few other minor things I noticed.

Let me know if there's anything else you'd like changed.